frontend/src/components/StudioCanvas.vue

```diff
@@ -278,6 +320,53 @@ onMounted(() => {
 	setPanAndZoom(canvasProps, canvasEl, canvasContainerEl)
 })
 
+function zoomIn() {
```

Comment:
1. Builder uses cmd + & cmd - to do so. Those are the standard zoom in & out shortcuts and we should stick to it.
2. Code for keyboard shortcuts is in `useStudioEvents` composable. So it should be there. Copy builder's logic instead of redoing it here.